src/elm/Page/Device.elm

```diff
@@ -895,6 +928,31 @@ renderConnectionStatus device =
                 ]
 
 
+buttonsRow : Bool -> Html Msg
+buttonsRow deviceCredentialsInhibited =
+    Grid.row []
+        [ Grid.col
+            [ Col.sm12
+            , Col.attrs [ Flex.block, Flex.rowReverse ]
+            ]
+            [ if deviceCredentialsInhibited then
+                Button.button
+                    [ Button.success
+                    , Button.onClick (SetCredentialsInhibited False)
+                    ]
+                    [ Html.text "Restore credentials" ]
```

Comment:
"Enable credentials"

Reply:
Probably "Enable credentials request" is more correct